### PR TITLE
Add hammer-cli-foreman-google-0.0.1

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

File: dependencies/bullseye/hammer_cli_foreman_google/control [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Source: ruby-hammer-cli-foreman-google
+Section: ruby
+Priority: optional
+Maintainer: Debian Ruby Extras Maintainers <pkg-ruby-extras-maintainers@lists.alioth.debian.org>
+Uploaders: Oleh Fedorenko <ofedoren@redhat.com>
```
Comment (evgeni) [inline dependencies/bullseye/hammer_cli_foreman_google/control:5]:
```suggestion
Maintainer: Oleh Fedorenko <ofedoren@redhat.com>
```

File: dependencies/bullseye/hammer_cli_foreman_google/control [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Build-Depends: debhelper-compat (= 12),
+               gem2deb (>= 1),
+               ruby-hammer-cli-foreman (>= 3.4.0)
+Standards-Version: 4.5.0
+Vcs-Git: https://salsa.debian.org/ruby-team/ruby-hammer-cli-foreman-google.git
+Vcs-Browser: https://salsa.debian.org/ruby-team/ruby-hammer-cli-foreman-google
```
Comment (evgeni) [inline dependencies/bullseye/hammer_cli_foreman_google/control:11]:
```suggestion
```

File: dependencies/bullseye/hammer_cli_foreman_google/salsa-ci.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (evgeni) [inline dependencies/bullseye/hammer_cli_foreman_google/salsa-ci.yml:1]: this file can be removed, we don't use Debian's CI system

**Review by evgeni — CHANGES_REQUESTED** [review]
some minor comments, as we don't use Debian's infrastructure, otherwise LGTM

(I commented only on bullseye, but the changes apply equally to both)

**Comment by ofedoren** [comment]
Thanks, @evgeni, updated.

**Review by evgeni — APPROVED** [review]

**Pull request merged**